$OpenBSD$

Needs ggrep for -P option but let's not bother with that.

Index: CMakeLists.txt
--- CMakeLists.txt.orig
+++ CMakeLists.txt
@@ -63,7 +63,7 @@ if(EXISTS "${CMAKE_SOURCE_DIR}/.git")
       OUTPUT_STRIP_TRAILING_WHITESPACE
     )
   endif()
-elseif(UNIX AND NOT APPLE)
+elseif(UNIX AND NOT APPLE AND NOT ${CMAKE_SYSTEM_NAME} MATCHES "OpenBSD")
   # Fallback for Ubuntu/Launchpad (extracts Git hash from debian/changelog rather than Git repo)
   # (see https://answers.launchpad.net/launchpad/+question/678556)
   execute_process(COMMAND sh -c "grep -Po '(?<=-)(([a-z0-9])\\w+)(?=\\+)' -m 1 changelog"
